src/components/Fingerprint.tsx

```diff
@@ -0,0 +1,68 @@
+import { Color } from "../types";
+
+interface Props {
+  width: number;
+  height: number;
+  color: Color;
+  top: number;
+  left: number;
+  hide?: boolean;
+}
+
+export function Fingerprint({ width, height, color, top, left, hide }: Props) {
+  // TODO: try to make this performance better. things to try
+  // - https://www.crmarsh.com/svg-performance/
+  // - use a div with a blur
+  // - What about image for the blob/fingerprint and then a layer with background-blend-mode with a specific colour
+  // try encoding the svg as an image source?
+  return (
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
+      version="1.1"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      {...{ "xmlns:svgjs": "http://svgjs.dev/svgjs" }}
+      viewBox={`0 0 ${width * 4} ${height * 4}`}
+      className="activeFingerprint"
+      width={width * 2}
+      height={height * 2}
+      style={{
+        top: `${top - height}px`,
+        left: `${left - width}px`,
+        display: hide ? "none" : "block",
```

Comment:
I think we should use `visibility` and `opacity` properties so we can animate it in/out

Reply:
ok sg